### PR TITLE
Contact Us Form

### PR DESCRIPTION
**Pull request by lili2311**
[Jira: PIMP-87](https://womenhackfornonprofits.atlassian.net/browse/PIMP-87)
Created initial reusable contact form.
screen shot 2016-02-14 at 03 20 28

### CODE REVIEW

**Comment by tanyapowell** [comment]
Submit button isn't centered :)

File: frontend/src/pages/create-account.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                        <label for="fname">First name:</label>
+                        <input type="text" name="fname" placeholder="First Name" required>
+                    </div>
+                    <div class="small-12 large-6 column">
+                        <label for="sname">Surname name:</label>
+                        <input type="text" name="sname" placeholder="Surname" required>
+                    </div>
+                    <div class="small-12 large-6 column">
+                        <label for="email">Email:</label>
+                        <input type="email" name="email" placeholder="Email" required>
```
Comment (tanyapowell) [inline frontend/src/pages/create-account.html:22]: For consistency, whatever the placeholder for the login.html `Email` field should be the same here. [See PR #5](url) 
image

File: frontend/src/pages/includes/contact.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            <label for="fname">First name:</label>
+            <input type="text" name="fname" placeholder="First Name" required>
+        </div>
+        <div class="small-12 large-6 column">
+            <label for="sname">Surname name:</label>
+            <input type="text" name="sname" placeholder="Surname" required>
+        </div>
+        <div class="small-12 large-6 column">
+            <label for="email">Email:</label>
+            <input type="email" name="_replyto" placeholder="Email" required>
```
Comment (tanyapowell) [inline frontend/src/pages/includes/contact.html:15]: See comment above

File: frontend/src/pages/includes/footer.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
-<footer class="row pnl-footer">
```
Comment (tanyapowell) [inline frontend/src/pages/includes/footer.html:1]: Is this the same change from [PR #5](url)?

**Comment by lili2311** [comment]
Created an [Issue](https://github.com/womenhackfornonprofits/pimpmycause-rebuild/issues/11)

**Pull request merged**